.eslintrc.json

```diff
@@ -10,7 +10,9 @@
         "@typescript-eslint/no-explicit-any": "off",
         "@typescript-eslint/no-this-alias": "off",
         "@typescript-eslint/no-empty-function": "off",
-        "@typescript-eslint/no-empty-interface": "off"
+        "@typescript-eslint/no-empty-interface": "off",
+        "@typescript-eslint/no-inferrable-types": "off",
```

Comment:
What's the reasoning for this one? I don't really see an issue with extra typing.

Reply:
It's about class prop initial number value:
```
class Foo {
  num1 // it's fine
  num2 // cause ESLint error
}
```